src/katsdpcontroller/generator.py

```diff
@@ -916,6 +916,15 @@ def _make_fgpu(
                     fgpu.sensor_renames[
                         f"{stream.name}.input{j}.{name}"
                     ] = f"{stream.name}.{label}.{name}"
+                for name in [
+                    "dither-seed",
+                ]:
+                    # These engine-level sensors get replicated as well, but are not per-input
+                    orig_sensor_name = f"{stream.name}.{name}"
+                    renames = fgpu.sensor_renames.get(orig_sensor_name, [])
+                    renames.append(f"{stream.name}.{label}.{name}")
+                    fgpu.sensor_renames[orig_sensor_name] = renames
```

Comment:
As noted in the general discussion I'd rather just add the right value to sensor_renames once rather than iteratively appending. But as a general tip, you can do this more simply like this (untested so I may have the syntax slightly wrong):
```suggestion
                    # These engine-level sensors get replicated as well, but are not per-input
                    fgpu.sensor_renames[f"{stream.name}.{name}"].setdefault([]).append(f"{stream.name}.{label}.{name}")
```

Reply:
Fair enough. I always seem to go the wrong way on these 50/50s.